tests/multi-threaded/test_quality_estimator.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#####################################################################
+# SUMMARY: Run tests for service-cli
+# AUTHOR: jerinphilip
+# TAGS: full, native
+#####################################################################
+
+set -eo pipefail;
+
+# Generate output specific to hardware.
+OUTFILE=$BRT_DATA/simple/bergamot/$(brt_outfile "quality_estimator")
+EXPECTED=$BRT_DATA/simple/bergamot/$(brt_expected "quality_estimator")
+
+${BRT_MARIAN}/bergamot-test --bergamot-mode test-quality-estimator ${BRT_FILE_ARGS} < ${BRT_DATA}/simple/bergamot/input.txt > $OUTFILE
```

Comment:
The source corresponding to this, in bergamot-translator? Could you bump the submodule in your active PR to use this branch of BRT?